File: mathvar.zig
// mathvar.zig
const std = @import("std");

pub fn main() void {
    const a: i32 = 5;
    const b: i32 = 3;
    const sum: i32 = a + b;
    const diff: i32 = a - b;
    const prod: i32 = a * b;
    const quot: i32 = a / b;

    std.debug.print("5 + 3 = {}\n", .{sum});
    std.debug.print("5 - 3 = {}\n", .{diff});
    std.debug.print("5 * 3 = {}\n", .{prod});
    std.debug.print("5 / 3 = {}\n", .{quot});
}

File: mathvar.vb
' mathvar.vb
Module mathvar
    Sub Main()
        Dim a As Integer = 5
        Dim b As Integer = 3
        Dim sum As Integer = a + b
        Dim diff As Integer = a - b
        Dim prod As Integer = a * b
        Dim quot As Integer = a / b

        Console.WriteLine("5 + 3 = " & sum)
        Console.WriteLine("5 - 3 = " & diff)
        Console.WriteLine("5 * 3 = " & prod)
        Console.WriteLine("5 / 3 = " & quot)
    End Sub
End Module